addons/pos_self_order/controllers/orders.py

```diff
@@ -224,7 +224,8 @@ def _process_lines(self, lines, pos_config, pos_order_id, take_away=False):
             product = pos_config.env['product.product'].browse(int(line.get('product_id')))
             lst_price = pricelist._get_product_price(product, quantity=line_qty) if pricelist else product.lst_price
             selected_attributes = fetched_attributes.browse(line.get('attribute_value_ids', []))
-            lst_price += sum([attr.price_extra for attr in selected_attributes])
+            price_extra = sum(attr.price_extra for attr in selected_attributes)
+            lst_price += price_extra
```

Comment:
Same here

Reply:
It's used here aswell, it's just to avoid recomputing the `sum(...)` https://github.com/odoo/odoo/pull/163401/files#diff-ffdbb901f270e360bd0cf7a7480a0731f97486d946763fefa0dfdbb21f3c9008R270